features/raadpleeg-verblijfplaats-met-peildatum/gba/test/foutsituaties-gba.feature

```diff
@@ -0,0 +1,312 @@
+      | naam                              | waarde                |
+      | type                              | RaadpleegMetPeildatum |
+      | burgerservicenummer               | 000000012             |
+      | peildatum                         | 2020-01-01            |
+      | exclusiefVerblijfplaatsBuitenland | <ongeldige waarde>    |
+      Dan heeft de response een object met de volgende gegevens
+      | naam     | waarde                                                            |
+      | type     | https://datatracker.ietf.org/doc/html/rfc7231#section-6.5.1       |
+      | title    | Een of meerdere parameters zijn niet correct.                     |
+      | status   | 400                                                               |
+      | detail   | De foutieve parameter(s) zijn: exclusiefVerblijfplaatsBuitenland. |
+      | code     | paramsValidation                                                  |
+      | instance | /haalcentraal/api/brphistorie/verblijfplaatshistorie              |
+      En heeft het object de volgende 'invalidParams' gegevens
+      | code    | name                              | reason                  |
+      | boolean | exclusiefVerblijfplaatsBuitenland | Waarde is geen boolean. |
+
+      Voorbeelden:
+      | ongeldige waarde |
+      | 0                |
+      | 1                |
+      | waar             |
+      | ja               |
+      | null             |
+      | undefined        |
+  
+  Rule: Alleen gespecificeerde parameters bij het opgegeven raadpleeg type mogen worden gebruikt
+
+    @fout-case
+    Abstract Scenario: De <parameter> parameter is gebruikt bij RaadpleegMetPeildatum
+      Als gba verblijfplaatshistorie wordt gezocht met de volgende parameters
+      | naam                | waarde                |
+      | type                | RaadpleegMetPeildatum |
+      | burgerservicenummer | 000000012             |
+      | peildatum           | 2020-01-01            |
+      | <parameter          | <waarde>              |
+      Dan heeft de response een object met de volgende gegevens
+      | naam     | waarde                                                      |
+      | type     | https://datatracker.ietf.org/doc/html/rfc7231#section-6.5.1 |
+      | title    | Een of meerdere parameters zijn niet correct.               |
+      | status   | 400                                                         |
+      | detail   | De foutieve parameter(s) zijn: <parameter>.                 |
+      | code     | paramsValidation                                            |
+      | instance | /haalcentraal/api/brphistorie/verblijfplaatshistorie        |
+      En heeft het object de volgende 'invalidParams' gegevens
+      | code         | name        | reason                      |
+      | unknownParam | <parameter> | Parameter is niet verwacht. |
+
+      Voorbeelden:
+      | parameter | peildatum  |
+      | datumVan  | 2019-01-01 |
```

Comment:
Wat is de reden om meerdere keren op dezelfde parameter naam te valideren? Een andere waarde heeft volgens mij geen invloed op de input validatie

Reply:
zou niet moeten inderdaad. test is juist om vast te stellen dat het niet uitmaakt. dus of de datumVan bijvoorbeeld ervoor (dan valt peildatum mogelijk binnen periode) of gelijk is, of erna ligt (peildatum ligt dan buiten de periode), in al die gevallen moet dezelfde foutmelding komen.